game-core/src/main/java/games/strategy/triplea/ai/AiPoliticalUtils.java

```diff
@@ -84,13 +84,15 @@ private static boolean wantToPerFormActionTowardsWar(final PoliticalActionAttach
   private static boolean goesTowardsWar(final PoliticalActionAttachment nextAction, final PlayerID p0,
       final GameData data) {
     for (final String relationshipChangeString : nextAction.getRelationshipChange()) {
-      final String[] relationshipChange = relationshipChangeString.split(":");
-      final PlayerID p1 = data.getPlayerList().getPlayerId(relationshipChange[0]);
-      final PlayerID p2 = data.getPlayerList().getPlayerId(relationshipChange[1]);
+      final PoliticalActionAttachment.RelationshipChange relationshipChange =
+          PoliticalActionAttachment.parseRelationshipChange(relationshipChangeString);
+      final PlayerID p1 = data.getPlayerList().getPlayerId(relationshipChange.player1Name);
+      final PlayerID p2 = data.getPlayerList().getPlayerId(relationshipChange.player2Name);
       // only continue if p1 or p2 is the AI
       if (p0.equals(p1) || p0.equals(p2)) {
         final RelationshipType currentType = data.getRelationshipTracker().getRelationshipType(p1, p2);
-        final RelationshipType newType = data.getRelationshipTypeList().getRelationshipType(relationshipChange[2]);
+        final RelationshipType newType =
+            data.getRelationshipTypeList().getRelationshipType(relationshipChange.relationshipTypeName);
```

Comment:
Same thought as player names and having this return `RelationshipType` instead of `String`?

Reply:
Same answer as above with respect to player names.  I'll consider this during the same prototype.